notes/stories/film/film-scifi-1990s.st.txt

```diff
@@ -11669,6 +11669,47 @@ factions joining forces against a common enemy [The people of Naboo joined force
 telekinetic ability [Obi-Wan used the power of his own mind to move a light saber to his hand in order to defeat Darth Maul.]
 
 
+movie: Deep Blue Sea (1999)
+===========================
+
+:: Title
+Deep Blue Sea
+
+:: Description
+Set in an isolated underwater facility, the film follows a team of scientists
+and their research on mako sharks to help fight Alzheimer's disease. The
+situation plunges into chaos when multiple genetically engineered sharks go
+on a rampage and flood the facility.
+
+:: Date
+1999-07-28
+
+:: References
+https://en.wikipedia.org/wiki/Deep_Blue_Sea_(1999_film)
+
+:: Ratings
+2 <paul>
+
+:: Genre
+sci-fi
+action
+
+:: Choice Themes
+human vs. beast [The people trapped in the ocean facility were being terrorized by multiple genetically engineered sharks.]
+
+:: Major Themes
+cure for an as yet uncurable disease [Dr. Susan McCallister was genetically engineering mako sharks to have bigger brains as part of her research to help cure Alzheimer's disease.]
+the dangers of unfettered scientific advancement [Dr. Susan McCallister accidentally created a pack of intelligent sharks that swam amok while seeking a cure for Alzheimer's disease.]
+species uplifting [The genetically engineered mako sharks made a complex plan to escape to the open ocean from the facility that was their prison. In particular, the sharks manipulated people into flooding the ocean facility, so they (the sharks) could escape to through some weak mesh fences at the surface, bypassing the lower titanium walls.]
```

Comment:
We should consider how this relates to "intelligent animal". Same thing except one is supposedly naturally smart and the other for some help?

Reply:
That's just the conclusion I arrived at after considering which, if any, of those two themes to apply here. Anything to update about the definition in your view?